### PR TITLE
app-emulation/wine-staging tidyups

### PR DESCRIPTION
**Pull request by bobwya**

### CODE REVIEW

File: app-emulation/wine-staging/wine-staging-9999.ebuild [hunk 1]
```diff
@@ -314,19 +317,18 @@ pkg_setup() {
 
 src_unpack() {
 	if [[ ${PV} == "9999" ]] ; then
-		EGIT_CHECKOUT_DIR="${S}" EGIT_COMMIT="${WINE_COMMIT}" git-r3_src_unpack
+		EGIT_CHECKOUT_DIR="${S}" git-r3_src_unpack
 		if use staging; then
 			local CURRENT_WINE_COMMIT=${EGIT_VERSION}
```
Comment (sarnex) [inline app-emulation/wine-staging/wine-staging-9999.ebuild]: Does this check work as expected with EGIT_OVERRIDE_COMMIT_WINE? As in, will you get the warning if you set EGIT_OVERRIDE_COMMIT_WINE to an unsupported version?

File: app-emulation/wine-staging/wine-staging-2.0-r1.ebuild [hunk 2]
```diff
@@ -266,14 +266,17 @@ wine_env_vcs_vars() {
 		if use staging; then
 			eerror "Because of the multi-repo nature of ${MY_PN}, ${pn_live_var}"
 			eerror "cannot be used to set the commit. Instead, you may use the"
-			eerror "environmental variables WINE_COMMIT, and STAGING_COMMIT."
+			eerror "environment variables:"
+			eerror "  EGIT_OVERRIDE_COMMIT_WINE"
+			eerror "  EGIT_OVERRIDE_COMMIT_WINE_STAGING_WINE_STAGING"
```
[the remark this answers is not in the document]
Comment (bobwya) [reply app-emulation/wine-staging/wine-staging-2.0-r1.ebuild]: @sarnex 
I've fixed the references that should be to: EGIT_OVERRIDE_COMMIT_WINE_COMPHOLIO_WINE_STAGING

[the remark this answers is not in the document]
Comment (bobwya) [reply app-emulation/wine-staging/wine-staging-2.0-r1.ebuild]: @sarnex 
I've fixed the references that should be to: EGIT_OVERRIDE_COMMIT_WINE_COMPHOLIO_WINE_STAGING

[the remark this answers is not in the document]
Comment (bobwya) [reply app-emulation/wine-staging/wine-staging-2.0-r1.ebuild]: @sarnex 
I've fixed the references that should be to: EGIT_OVERRIDE_COMMIT_WINE_COMPHOLIO_WINE_STAGING

[the remark this answers is not in the document]
Comment (bobwya) [reply app-emulation/wine-staging/wine-staging-2.0-r1.ebuild]: @sarnex 
I've fixed the references that should be to: EGIT_OVERRIDE_COMMIT_WINE_COMPHOLIO_WINE_STAGING

[the remark this answers is not in the document]
Comment (bobwya) [reply app-emulation/wine-staging/wine-staging-2.0-r1.ebuild]: @sarnex 
I've fixed the references that should be to: EGIT_OVERRIDE_COMMIT_WINE_COMPHOLIO_WINE_STAGING

[the remark this answers is not in the document]
Comment (bobwya) [reply app-emulation/wine-staging/wine-staging-2.0-r1.ebuild]: @sarnex 
I've fixed the references that should be to: EGIT_OVERRIDE_COMMIT_WINE_COMPHOLIO_WINE_STAGING

**Comment by bobwya** [comment]
@sarnex 
I've fixed the references that should be to: EGIT_OVERRIDE_COMMIT_WINE_COMPHOLIO_WINE_STAGING

File: app-emulation/wine-staging/wine-staging-2.0-r1.ebuild (hunk 2 above)
[the remark this answers is not in the document]
Comment (bobwya) [reply app-emulation/wine-staging/wine-staging-2.0-r1.ebuild]: @sarnex 
I've fixed the references that should be to: EGIT_OVERRIDE_COMMIT_WINE_COMPHOLIO_WINE_STAGING

[the remark this answers is not in the document]
Comment (bobwya) [reply app-emulation/wine-staging/wine-staging-2.0-r1.ebuild]: @sarnex 
I've fixed the references that should be to: EGIT_OVERRIDE_COMMIT_WINE_COMPHOLIO_WINE_STAGING

[the remark this answers is not in the document]
Comment (bobwya) [reply app-emulation/wine-staging/wine-staging-2.0-r1.ebuild]: @sarnex 
I've fixed the references that should be to: EGIT_OVERRIDE_COMMIT_WINE_COMPHOLIO_WINE_STAGING

**Pull request closed without merging**